src/utils/Constants.tsx

```diff
@@ -78,3 +79,346 @@ export const NAV_ITEMS = [
     nestedItems: [],
   },
 ]
+
+export const TABLE_DATA_HEADERS: column[] = [
+  { id: 1, value: '', type: 'checkbox', alignText: 'center' },
+  {
+    id: 2,
+    value: 'Journey Name',
+    type: 'sort',
+    alignText: 'center',
+  },
+  {
+    id: 3,
+    value: 'Description',
+    type: 'text',
+    alignText: 'center',
+  },
+  { id: 4, value: 'Group', type: 'sort', alignText: 'center' },
+  { id: 5, value: 'Status', type: 'sort', alignText: 'center' },
+]
+
+
+
+export const TABLE_ROWS_DATA = [
```

Comment:
Need to work on Table Data. 
Will share some sample data

Reply:
Okay @inigorajv